main.go

```diff
@@ -2,18 +2,25 @@ package main
 
 import (
 	"context"
+	"log"
 	"net/http"
 	"os"
 	"os/signal"
 	"ship-it/internal/api"
+	"ship-it/internal/ecrconsumer"
 	"syscall"
+	"time"
 
-	"github.com/go-kit/kit/log"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	kitlog "github.com/go-kit/kit/log"
```

Comment:
We won't have to do this qualified import once we remove the usage of the stdlib `log` pkg.